native_client/definitions.mk

```diff
@@ -38,8 +40,48 @@ AS      := $(TOOLCHAIN)as
 CC      := $(TOOLCHAIN)gcc
 CXX     := $(TOOLCHAIN)c++
 LD      := $(TOOLCHAIN)ld
+LDD     := $(TOOLCHAIN)ldd $(TOOLCHAIN_LDD_OPTS)
 
+RPATH_PYTHON         := '-Wl,-rpath,\$$ORIGIN/lib/'
+RPATH_NODEJS         := '-Wl,-rpath,$$\$$ORIGIN/../../../'
 META_LD_LIBRARY_PATH := LD_LIBRARY_PATH
 ifeq ($(OS),Darwin)
 META_LD_LIBRARY_PATH := DYLD_LIBRARY_PATH
+RPATH_PYTHON         := '-Wl,-rpath,@loader_path/lib/'
+RPATH_NODEJS         := '-Wl,-rpath,@loader_path/../../../'
 endif
+
+define copy_missing_libs
```

Comment:
Please add a line documenting what this function does.

Reply:
Done